tests/end_to_end/andor_assignment/irix-o2-out.c

```diff
@@ -0,0 +1,16 @@
+s32 test(s32 arg0, s32 arg1, s32 arg2, s32 arg3)
+{
+    s32 sp1C;
+    s32 temp_a0;
+    s32 phi_v1;
+
+    if (((((arg0 + arg1) != 0) || temp_a0 = arg1 + arg2, (temp_a0 != 0)) || sp1C = 0, (func_00400090(temp_a0) != 0)) || phi_v1 = sp1C, (arg3 != 0))
```

Comment:
this is one terrifying conditional... but it checks out

Reply:
yep...